example/Vagrantfile

```diff
@@ -5,100 +5,134 @@
         end
         target.vm.network "forwarded_port", guest: 8080, host: 9090
         target.vm.network "forwarded_port", guest: 5672, host: 9091
         target.vm.network "forwarded_port", guest: 9080, host: 9092
         target.vm.network "forwarded_port", guest: 8443, host: 9093
-
-        # NOTE: virtualbox shared folder syncing can be quite slow, but shared folders
-        # can be configured as NFS mounts to the host instead.
-        # To do so, append: `, type: "nfs"` to all of the target.vm.synced_folder commands
-        # below, then on the host system, run `sudo nfsd start`.
-        # Additionally, uncomment the below configuration:
-        #
-        # For NFS mounts, a host only network is needed. Feel free to change the IP.
-        # target.vm.network "private_network", ip: "192.168.50.4"
-
+        #usc-service
+        target.vm.network "forwarded_port", guest: 7080, host: 7080
+        
         # If true, then any SSH connections made will enable agent forwarding.
         # Default value: false
         target.ssh.forward_agent = true
 
         target.vm.provision "shell", inline: <<-SHELL
-          sudo service isc-dhcp-server start
-        #   service rsyslog stop
-        #   echo manual | sudo tee /etc/init/rsyslog.override
+          timeout=0
+          maxto=30
+          waitForPM2Daemon() {
+            while [ ${timeout} != ${maxto} ]; do
+              ps aux | grep PM2 | grep Daemon
+              if [ $? = 0 ]; then 
+                break
+              fi
+              sleep 1
+              timeout=`expr ${timeout} + 1`
+            done
+            if [ ${timeout} == ${maxto} ]; then
+              echo "Timed out waiting for PM2 Daemon (duration=`expr $maxto \* 1`s)."
+              exit 1
+            fi
+          }
+          service isc-dhcp-server start
+          service rsyslog stop
+          echo manual | sudo tee /etc/init/rsyslog.override
+          if [ "#{ENV['MULTI']}" ]; then
+            pm2 logs > /home/vagrant/src/build/vagrant.log &
+          else
+            pm2 logs > /home/vagrant/src/"#{ENV['REPO_NAME']}"/vagrant.log &
+          fi
+          waitForPM2Daemon
+          pm2 start rackhd-pm2-config.yml
         SHELL
-    end
-
-    # Virtual HW SERVER
-    config.vm.define "quanta_d51" do |target|
-        target.vm.box = "InfraSIM/quanta_d51"
-        target.vm.provider "virtualbox" do |v|
-            v.gui = true
-            v.memory = 4096
-            v.cpus = 4
-            v.customize ["modifyvm", :id, "--nicpromisc2", "allow-all", "--ioapic", "on"]
-        end
-        #target.vm.network :public_network
-        target.vm.network "private_network", virtualbox__intnet: "closednet", auto_config: false, nic_type: "82540EM"
-        target.vm.provision "shell", inline: <<-SHELL
-            sudo ifconfig eth1 up
-            sleep 2
-            sudo udhcpc -i br1
-        SHELL
-    end
```

Comment:
del of 83-101, this removes the creation of our HW simulator... this should not be removed or there will be no way to demo the functioning of this system.